app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def self.update_post_likes_counter(post)
+    post.update(likes_counter: Like.where(post_id: post.id).length)
+  end
```

Comment:
- Please remove the parameter `post` from the update_post_likes_counter method. You already defined the `post` in line 3.
- A faster way to update the likes_counter would be:
```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```